security/certs_test.go

```diff
@@ -31,7 +31,7 @@ import (
 func TestGenerateCerts(t *testing.T) {
 	// Do not mock cert access for this test.
 	security.ResetReadFileFn()
-	defer security.ResetTest()
+	defer security.SetReadFileFn(securitytest.Asset)
```

Comment:
I'd even add something to `securitytest`:

```
func MockReadFileFn() {
  security.SetReadFileFn(Asset)
}
```
and then make that the only entry point for the mock.

Otherwise you have places like these where you're duplicating the mock.